sdk/maps/Azure.Maps.Weather/src/Models/Options/GetAirQualityHourlyForecastsOptions.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.GeoJson;
+using Azure.Maps.Weather;
+
+namespace Azure.Maps.Weather.Models.Options
+{
+    /// <summary> Options. </summary>
+    public class GetAirQualityHourlyForecastsOptions : WeatherBaseOptions
+    {
+        /// <summary> Specifies for how long the responses are returned. </summary>
+        public int? Duration { get; set; }
```

Comment:
For measurable class member (time/distance/length/temperature), we should always clarify the unit. I suggest to rename it to `DurationInHours`

Reply:
Same for other `Duration` member variable in other `Options` classes